Documentation/ExtensionArchitecture/Concepts/SystemAndLocalExtensions.rst

```diff
@@ -1,55 +1,59 @@
 .. include:: /Includes.rst.txt
 
-
 .. _extension-scope:
 
 ===============================
 System and Local Extensions
 ===============================
 
-The files for an extension are located in a folder named by the
-*extension key*. The location of this folder can be either inside
-`typo3/sysext/` or `typo3conf/ext/`.
-
-The extension *must* be programmed so that it does automatically
-detect where it is located and can work from both locations.
-
+The files for an extension are located in a folder named vendor, followed by
```

Comment:
This is not really true. For an integrator or developer the extensions are placed in a "packages" or "local_packages" folder, which then gets symlinked via Composer into vendor.

This sounds now, that I have to place my extensions directly under the "vendor" folder.

Reply:
Rphrased it